INSTALL

```diff
@@ -1,10 +1,9 @@
 Thanks for downloading Django.
 
 To install it, make sure you have Python 3.6 or greater installed. Then run
-this command from the command prompt:
+the following commands from the command prompt:
 
-    python setup.py install
-
-If you're upgrading from a previous version, you need to remove it first.
+    python -m ensurepip
```

Comment:
I'm going to drop this line because users shouldn’t need to invoke this module directly (see [documentation](https://docs.python.org/3/library/ensurepip.html)), moreover it can cause issues for new users and sometimes it's disabled  for the system python, e.g.
```
$ python -m ensurepip

ensurepip is disabled in Debian/Ubuntu for the system python.
...
```